doc/conf.py

```diff
@@ -169,12 +169,13 @@
     'navbar_class': "navbar",
     'bootstrap_version': "3",  # default
     'navbar_links': [
-        ("Install", "getting_started"),
+        ("Install", "install/overview"),
```

Comment:
As it's meant to be the landing/default page, I would now rename the file `install/index.rst` (that way someone navigating to `dev/install` will get taken to this landing page instead of a not-found one)